### PR TITLE
update: package updates

### LINKED ISSUE #163: Update NPM Packages to Latest Versions
### Issue Description

We need to update the npm packages in our project to ensure we are using the latest, most secure, and stable versions. This will help us avoid potential security vulnerabilities, improve performance, and take advantage of new features and bug fixes.

### Reproducibility

 - [ ] This issue is reproducible
 - [x] This issue is not reproducible

### Steps to Reproduce
1. 
2. 

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [x] Medium
 - [ ] Low

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by ashifkhn**
Date: `25th August, 2024`

Developer Name: @ashifkhn 

---

## Issue Ticket Number

- Closes #163 

## Description
Packages have been updated and volta is added

Package updates.

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

Screenshot 2024-08-25 at 5 02 41 PM
Screenshot 2024-08-25 at 5 02 45 PM

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

**Review by Achintya-Chatterjee — APPROVED** [review]

File: jest.setup.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import '@testing-library/jest-dom';
-import './setupTests';
+// import './setupTests';
```
Comment (vinit717) [inline jest.setup.js:2]: why comment?

Comment (Achintya-Chatterjee) [reply jest.setup.js:2]: @vinit717 tests is not setup on members-site

Comment (ashifkhn) [reply jest.setup.js]: @vinit717 resolved. Will fix this in different pr.

Comment (vinit717) [reply jest.setup.js]: Can you mention the issue ticket for test fix

Comment (ashifkhn) [reply jest.setup.js]: I have made it draft for now, some test are failing.

Comment (ashifkhn) [reply jest.setup.js]: https://github.com/Real-Dev-Squad/members-site/pull/170

**Review by vinit717 — APPROVED** [review]

**Pull request merged**